src/internal.c

```diff
@@ -12913,6 +12939,30 @@ static int ProcessPeerCertParse(WOLFSSL* ssl, ProcPeerCertArgs* args,
     }
 #endif
 
+#ifdef WOLFSSL_MAXQ10XX_TLS
+    /* MAXQ10xx accepts ASN_NO_SIGNER_E because the CA cert is registered with
+     * the MAXQ hardware; not CertManager. All other return values are
+     * unexpected.
+     */
+    if (ret != ASN_NO_SIGNER_E) {
+        return ret;
+    }
+#endif /* WOLFSSL_MAXQ10XX_TLS */
+
+#ifdef HAVE_PK_CALLBACKS
```

Comment:
Please add a comment explaining what this block is doing.

Reply:
Fixed with 3b5a8330c